README.md

```diff
@@ -364,10 +372,10 @@ because we have to go through all the tests and ensure that they match
 the specification correctly. But we look at all of them, and take
 everything that we can.
 
-OWNERS files are used only to indicate who should be notified of pull
+`OWNERS` files are used only to indicate who should be notified of pull
```

Comment:
I think I'd leave these unquoted, like in docs/introduction.md.

Reply:
aye, understood